Templates/default/Config.stencil

```diff
@@ -0,0 +1,8 @@
+import ProjectDescription
+
+let config = Config(
+//    Create an account with "tuist auth" and a projet with "tuist project create"
```

Comment:
How about providing a link to the docs for getting started: https://docs.tuist.io/cloud/get-started.html

Reply:
Good idea. I'll use the new URL instead.